clang/include/clang/Driver/Options.td

```diff
@@ -8484,6 +8484,9 @@ def _SLASH_fp_strict : CLFlag<"fp:strict">, HelpText<"">, Alias<ffp_model_EQ>, A
 def _SLASH_fsanitize_EQ_address : CLFlag<"fsanitize=address">,
   HelpText<"Enable AddressSanitizer">,
   Alias<fsanitize_EQ>, AliasArgs<["address"]>;
+def : CLJoined<"fno-profile-sample-use">, Alias<fno_profile_sample_use>;
```

Comment:
Do we really need these aliases?

Reply:
For me, /fprofile-sample-use= is needed.
 I saw other CL _EQ flags has _COL alias.
In fact, I don't really need /fno-profile-sample-use.

Reply:
Removed /fno-profile-sample-use.